expat/xmlwf/xmlwf.c

```diff
@@ -215,8 +215,8 @@ endElement(void *userData, const XML_Char *name) {
 
 static int
 nsattcmp(const void *p1, const void *p2) {
-  const XML_Char *att1 = *(const XML_Char **)p1;
-  const XML_Char *att2 = *(const XML_Char **)p2;
+  const XML_Char *att1 = *(const XML_Char * const *)p1;
+  const XML_Char *att2 = *(const XML_Char * const *)p2;
```

Comment:
clang-format CI [asks us](https://github.com/libexpat/libexpat/actions/runs/3339947053/jobs/5529494973#step:6:128) to apply this change on top:
```diff
@@ -215,8 +215,8 @@ endElement(void *userData, const XML_Char *name) {
 
 static int
 nsattcmp(const void *p1, const void *p2) {
-  const XML_Char *att1 = *(const XML_Char * const *)p1;
-  const XML_Char *att2 = *(const XML_Char * const *)p2;
+  const XML_Char *att1 = *(const XML_Char *const *)p1;
+  const XML_Char *att2 = *(const XML_Char *const *)p2;
   int sep1 = (tcsrchr(att1, NSSEP) != 0);
   int sep2 = (tcsrchr(att2, NSSEP) != 0);
   if (sep1 != sep2)
```

Reply:
Done.